test/Spec.hs

```diff
@@ -151,6 +151,30 @@ mobileGenWith
   )
   ''Newtype2
 
+newtype Newtype3 = Newtype3 Text
+
+mobileGenWith
+  ( defaultOptions
+      { dataAnnotations = [JvmInline, Parcelize, Serializable],
```

Comment:
I think you should remove JvmInline here to ensure it is automatically added.

Reply:
Handled in https://github.com/MercuryTechnologies/moat/pull/8/commits/d1c3191e55140aad0061c1c9aee17f9426d1a885